frontend/controller/actions/identity.js

```diff
@@ -17,13 +19,25 @@ export default sbp('sbp/selectors/register', {
       console.warn(`deleting settings for pre-existing identity ${username}!`, oldSettings)
       await sbp('gi.db/settings/delete', username)
     }
+
+    let finalPicture = `${window.location.origin}/assets/images/user-avatar-default.png`
```

Comment:
If they don't provide a picture, would this be a good place to generate it instead of in the SignupForm?

Reply:
This is used in case, for some reason, `blockies` fail for some reason. That way we still have a very basic default avatar. This is called _graceful degradation_